WTWSMS/history/titles/c_alcantara.txt

```diff
@@ -9,12 +9,17 @@
 	holder=20381
 	liege = "k_visigoths"
 }
-480.1.1={
-	holder = 1301295 # Salla, Visigothic dux in Spain 483
```

Comment:
I think this also is not intended, then.

Reply:
Why did I change that? I don't understand